README.md

```diff
@@ -1,21 +1,35 @@
-Tox Client Guidelines
+# Tox Client Guidelines.
 ===
+This git repo contains the documentation for the suggestions and recommendations a model Tox client would follow. As
+with any client, you don't have to follow them as most are merely recommendation however if you do not only will your
+users will thank you, but other clients won't hate you either!
 
-This contains all the docs a model Tox client should follow. You don't have to follow them, these are just recommendation however if you do your users will thank you.
+Additionally, some client quarks are also detailed to make interaction a little enigmatic.
```

Comment:
shouldn't this be "quirks" (also at [L31](https://github.com/irungentoo/Tox_Client_Guidelines/pull/1/files#diff-04c6e90faac2675aa89e2176d2eec7d8R31))?